openfl/component/ca/ca.py

```diff
@@ -99,19 +104,24 @@ def get_token(name, ca_url, ca_path='.'):
     ])
 
 
-def get_ca_bin_paths(ca_path):
-    """Get paths of step binaries."""
-    ca_path = Path(ca_path)
-    step = None
-    step_ca = None
-    if (ca_path / 'step').exists():
-        dirs = os.listdir(ca_path / 'step')
-        for dir_ in dirs:
-            if 'step_' in dir_:
-                step = ca_path / 'step' / dir_ / 'bin' / 'step'
-            if 'step-ca' in dir_:
-                step_ca = ca_path / 'step' / dir_ / 'bin' / 'step-ca'
-    return step, step_ca
+def get_step_bin_path(ca_path: Union[Path, str]) -> Union[Path, None]:
+    """Get path of step binary."""
+    ca_path = Path(ca_path) / 'step'
+    step_path = None
+    if ca_path.exists():
+        ca_last_version = os.listdir(ca_path)[-1]
```

Comment:
`os.listdir` - "Return a list containing the names of the entries in the directory given by path. The list is in **arbitrary order**". 
It may not be the latest version.

Reply:
Fixed